database/factories/StorageProviderFactory.php

```
@@ -13,13 +17,53 @@ class StorageProviderFactory extends Factory
+                'app_secret' => 'test-app-secret',
+                'refresh_token' => 'test-refresh-token',
+            ],
+            S3::id() => [
+                'api_url' => 'https://s3.amazonaws.com',
+                'key' => 'test-key',
+                'secret' => 'test-secret',
+                'region' => 'us-east-1',
+                'bucket' => 'test-bucket',
+                'path' => '/backups',
+            ],
+            FTP::id(), SFTP::id() => [
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\StorageProviders\FTP' in method 'credentialsFor'. (undefined)

(StaticAccess)